Update ADFS PRT issue for 2019 (#6052)
MicrosoftDocs/windows-itpro-docs: windows/security/identity-protection/hello-for-business/hello-cert-trust-adfs.md

Prepare the Active Directory Federation Services deployment by installing and updating two Windows Server 2016 Servers.  Ensure the update listed below is applied to each server before continuing.    

> >[!NOTE] 

Reviewer: ```suggestion
> [!NOTE]
```

> > For AD FS 2010, if Windows Hello for Business with a Hybrid Certificate trust is performed, a known PRT issue exists. You may encounter this error in ADFS Admin event logs: Received invalid Oauth request. The client 'NAME' is forbidden to access the resource with scope 'ugs'. 

Reviewer 2: This should be AD FS 2019

Reviewer: May as well add a commit suggestion, since I noticed some MD codestyle I want to see improved:
```suggestion
> For AD FS 2019, if Windows Hello for Business with a Hybrid Certificate trust is performed, a known PRT issue exists. You may encounter this error in ADFS Admin event logs: Received invalid Oauth request. The client 'NAME' is forbidden to access the resource with scope 'ugs'. 
```

> To remediate this error: 

> > 1. Launch AD FS management console. Brose to "Services > Scope Descriptions"

Reviewer: ```suggestion
> 1. Launch AD FS management console. Browse to **Services** > **Scope Descriptions**
```

> > 2. Right click "Scope Descriptions" and select "Add Scope Description"

Reviewer: ```suggestion
> 2. Right click **Scope Descriptions** and select **Add Scope Description**
```

> > 3. Under name type "ugs" and Click Apply > OK

Reviewer: ```suggestion
> 3. Under name type "ugs" and click **Apply** > **OK**
```

> 4. Launch Powershell as Administrator

> > 5. Execute the command "Get-AdfsApplicationPermission". Look for the ScopeNames :{openid, aza} that has the ClientRoleIdentifier. Make a note of the ObjectIdentifier.

Reviewer: ```suggestion
> 5. Execute the command `Get-AdfsApplicationPermission`. Look for the ScopeNames :{openid, aza} that has the ClientRoleIdentifier. Make a note of the ObjectIdentifier.
```

> > 6. Execute the command "Set-AdfsApplicationPermission -TargetIdentifier <ObjectIdentifier from step 5> -AddScope 'ugs'

Reviewer: ```suggestion
> 6. Execute the command `Set-AdfsApplicationPermission -TargetIdentifier <ObjectIdentifier from step 5> -AddScope 'ugs'`
```

> 7. Restart the ADFS service.
> 8. On the client: Restart the client. User should be prompted to provision WHFB.

> > 9. If the provisioning window does not pop up then need to collect NGC trace logs and further troubleshoot.

Reviewer: ```suggestion
> 9. If the provisioning window does not pop up, you need to collect NGC trace logs and perform further troubleshooting.
```

> 

Reviewer: Redundant blank line. 2 blank lines are plenty sufficient as spacing between paragraphs and may not even translate to the same amount of blank lines on the target page.
```suggestion
```
